Essentials/src/main/java/com/earth2me/essentials/commands/Commandcreatekit.java

```diff
@@ -71,21 +63,27 @@ public void run(final Server server, final User user, final String commandLabel,
             ess.getKits().addKit(kitname, list, delay);
             user.sendMessage(tl("createdKit", kitname, list.size(), delay));
         } else {
-            final ConfigurationSection config = new MemoryConfiguration();
-            config.set("kits." + kitname + ".delay", delay);
-            config.set("kits." + kitname + ".items", list);
-
-            final Yaml yaml = new Yaml(yamlConstructor, yamlRepresenter, yamlOptions);
-            String fileContents = "# Copy the kit code below into the kits section in your config.yml file\n";
-            fileContents += yaml.dump(config.getValues(false));
-
-            uploadPaste(user.getSource(), kitname, delay, fileContents);
+            uploadPaste(user.getSource(), kitname, delay, list);
         }
     }
 
-    private void uploadPaste(final CommandSource sender, final String kitName, final long delay, final String contents) {
+    private void uploadPaste(final CommandSource sender, final String kitName, final long delay, final List<String> list) {
         executorService.submit(() -> {
             try {
+                final ConfigurationNode config = BasicConfigurationNode.root();
+                config.node("kits", kitName, "delay").set(delay);
+                config.node("kits", kitName, "items").set(list);
+
+                final StringWriter sw = new StringWriter();
+
+                final YamlConfigurationLoader loader = YamlConfigurationLoader.builder().sink(() -> new BufferedWriter(sw)).indent(2).nodeStyle(NodeStyle.BLOCK).build();
+                loader.save(config);
+
+                String fileContents = "# Copy the kit code below into the kits section in your config.yml file\n";
+                fileContents += sw.toString();
```

Comment:
```suggestion
                sw.append("# Copy the kit code below into the kits section in your config.yml file\n");
                String fileContents = sw.toString();
                loader.save(config);
```

(nuke the flush and close too, they're no-ops)